test/jobs/calendar_importer/calendar_importer_task_test.rb

```diff
@@ -189,4 +189,26 @@ class CalendarImporterTaskTest < ActiveSupport::TestCase
       assert_equal "Source responded with invalid JSON (783: unexpected token at '{ \"key\": \"va')", error.message
     end
   end
+
+  test 'can import from generic iCal feed' do
+    VCR.use_cassette(:generic_ical_feed, allow_playback_repeats: true) do
+      calendar = create(
+        :calendar,
+        name: 'Generic iCal Calendar',
+        source: 'https://www.birchcommunitycentre.co.uk/events.ics',
```

Comment:
can we get tests for the `http` and `webcal` type links that got added (I'm assuming) after this test was written.

Reply:
I put in a test for webcal. i can put in *a* test for `http` but to be thorough and check all the patterns i'd have to write an `http` variant for every type of ical feed we support.

Reply:
if that is something that would be valuable then let me know